.github/actions/build/action.yml

```diff
@@ -61,6 +61,7 @@ runs:
         NEXT_PUBLIC_TRANSAK_APP_URL: 'https://global.transak.com'
         NEXT_PUBLIC_TRANSAK_API_URL: 'https://api.transak.com'
         NEXT_PUBLIC_TRANSAK_API_KEY: '799087ea-c2be-44b8-a0fe-56d96b37ee3c'
+        NEXT_PUBLIC_TURNSTILE_SITE_KEY: 0x4AAAAAAABe-kMUW_1JUfT7
         NEXT_PUBLIC_FORK_BASE_CHAIN_ID: '${{ inputs.NEXT_PUBLIC_FORK_BASE_CHAIN_ID }}'
         NEXT_PUBLIC_FORK_CHAIN_ID: '${{ inputs.NEXT_PUBLIC_FORK_CHAIN_ID }}'
         NEXT_PUBLIC_FORK_URL_RPC: '${{ inputs.NEXT_PUBLIC_FORK_URL_RPC }}'
```

Comment:
@MareskoY I imagine we'd need to update for this new env var in cypress and maybe write a test for this new feature?

Reply:
I'm not sure that we want to test this with cypress, I think it'd just be a waste of eth in our relayer.

Reply:
current test coverage never touches the faucet